src/test/scala/mesosphere/marathon/core/launcher/impl/OfferProcessorImplTest.scala

```diff
@@ -75,7 +75,7 @@ class OfferProcessorImplTest extends MarathonSpec {
     Await.result(offerProcessor.processOffer(offer), 1.second)
 
     verify(offerMatcher).matchOffer(deadline, offer)
-    verify(taskLauncher).declineOffer(offerId)
+    verify(taskLauncher).declineOffer(offerId, None)
```

Comment:
This test would be more fun if we actually also tested with a custom decline duration.